scripts/experiments_tabular/knn.jl

```diff
@@ -51,6 +51,7 @@ function fit(data, parameters)
 		global info, fit_t, _, _, _ = @timed fit!(model, data[1][1])
 	catch e
 		# return an empty array if fit fails so nothing is computed
+		rethrow(e)
```

Comment:
I suppose this has been added to stop training of other seeds, if it happens to fail on one. Am I right?

Reply:
Actually this was part of debugging, it should not be there, because it will stop the training completely.